nailgun/entities.py

```diff
@@ -3019,6 +3019,7 @@ def __init__(self, server_config=None, **kwargs):
                 default='yum',
                 required=True,
             ),
+            'container_repository_name': entity_fields.StringField(),
```

Comment:
This field is needed to create docker containers with custom content view, lifecycle environment etc.
I'll submit a test which uses it later today, and for now just a little proof this field is ok and working:
```python
>>> repo1 = entities.Repository(id=96).read()
>>> repo1.container_repository_name
u'fc6a2b53-17d5-4053-b607-7aa82437f1f4-933e9f32-24d5-4548-a676-8fe3c8913c8d-8fa2f93d-8dfc-4774-a692-444cbe6e30bd'
```